pkgs/applications/graphics/apitrace/default.nix

```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   name = "apitrace-${version}";
-  version = "7.1";
+  version = "7.1-git-20170411";
```

Comment:
This was taken from our package naming convention as it would refer to an unreleased package: https://nixos.org/nixpkgs/manual/#sec-package-naming